cms/plugin_rendering.py

```diff
@@ -18,6 +18,8 @@
 from cms.utils.django_load import iterload_objects
 from cms.utils.placeholder import get_toolbar_plugin_struct, restore_sekizai_context
 
+from cms.utils.permissions import has_plugin_permission
```

Comment:
please add this import under ``from cms.utils.django_load import iterload_objects`` to keep imports in alphabetical order.